### ISSUE TITLE
Sonargraph Integration throws error on Maven Sonar plugin execution

### ISSUE THREAD

**Issue by alan-czajkowski** [sonargraph/sonar-sonargraph-integration#20]
Using the Maven SonarQube plugin in my project:
```
<plugin>
  <groupId>org.sonarsource.scanner.maven</groupId>
  <artifactId>sonar-maven-plugin</artifactId>
  <version>3.7.0.1746</version>
</plugin>
```

When running `mvn sonar:sonar` on the project (in a Jenkins build), the following error occurs:
```
...
[2020-04-13T19:17:01.673Z] [INFO] Quality profile for xml: Sonar way
[2020-04-13T19:17:01.673Z] [INFO] ------------- Run sensors on module Example Application
[2020-04-13T19:17:01.941Z] [INFO] Load metrics repository
[2020-04-13T19:17:01.941Z] [INFO] Load metrics repository (done) | time=53ms
[2020-04-13T19:17:02.537Z] [INFO] Sonargraph Integration: Loaded standard metrics file '/com/hello2morrow/sonargraph/integration/sonarqube/SonargraphMetrics.properties'
[2020-04-13T19:17:02.537Z] [ERROR] Sonargraph Integration: Unable to load custom metrics file '/jenkins/cloudbees-core-oc/.sonargraphintegration/SonargraphMetrics.properties'
[2020-04-13T19:17:02.537Z] java.io.FileNotFoundException: /jenkins/cloudbees-core-oc/.sonargraphintegration/SonargraphMetrics.properties (No such file or directory)
[2020-04-13T19:17:02.537Z] 	at java.io.FileInputStream.open0(Native Method)
[2020-04-13T19:17:02.537Z] 	at java.io.FileInputStream.open(FileInputStream.java:195)
[2020-04-13T19:17:02.537Z] 	at java.io.FileInputStream.<init>(FileInputStream.java:138)
...
```

There is no Sonargraph plugin configuration in the pom.xml, nor am I interested in defining one -- is it required to be defined?

Is this a bug with the default behaviour, or does this require some kind of configuration?

**Comment by alan-czajkowski** [comment]
should it throw a WARN instead of ERROR? or is it a legitimate error?

**Comment by ingmarkellner** [comment]
Hi Alan,

thanks for letting us know. You are right, this should not be logged as an error. In https://github.com/sonargraph/sonar-sonargraph-integration/blob/master/src/main/java/com/hello2morrow/sonargraph/integration/sonarqube/SonargraphMetricsProvider.java, method loadCustomMetrics(), there is simply the check missing, if the file actually exists, before trying to load its contents.

No, it is not required to define this file. It will be automatically generated, if you have defined custom metrics via Sonargraph scripts or you decide to activate scripts that Sonargraph provides as part of its quality models. The documentation provides more details about that feature: https://eclipse.hello2morrow.com/doc/standalone/content/groovy_script_producing_results.html.

In short: You can ignore this error and I will publish a fix with the next release. A fix is already pushed to the develop branch.
